api/envoy/admin/v3/config_dump.proto

```diff
@@ -346,3 +347,36 @@ message SecretsConfigDump {
   // warming in preparation to service clusters or listeners.
   repeated DynamicSecret dynamic_warming_secrets = 3;
 }
+
+// [#not-implemented-hide:]
+// Envoy's EDS implementation *will* fill this message with all currently known endpoints. Endpoint
+// configuration information can be used to recreate an Envoy configuration by populating all
```

Comment:
Just a thought, since number of endpoints could potentially be very large, do we want to dump out all of them or should we have some upper limit.

Would like to see what the API reviewer thinks.

Reply:
For the admin handler, this could be just a query param. Could be worth adding as needed.